src/script/view_model/ListViewModel.ts

```diff
@@ -374,7 +374,7 @@ export class ListViewModel {
       if (customLabel) {
         entries.push({
           click: () => conversationLabelRepository.removeConversationFromLabel(customLabel, conversationEntity),
-          label: t('conversationsPopoverRemoveFrom', customLabel.name),
+          label: t('conversationsPopoverRemoveFrom', customLabel.name, {}, true),
```

Comment:
had to look that up :sweat_smile: 
For reference `{}` : passed to `dangerousSubstitutions` and `true`: passed to `skipEscape`